src/towncrier/_builder.py

```diff
@@ -53,6 +55,35 @@ def parse_newfragment_basename(
         return invalid
 
 
+class FragmentsPath:
```

Comment:
Is this expected to be public API?

I can do something like this: `from towncrier.create import FragmentsPath`